Dockerfile

```diff
@@ -32,10 +27,18 @@ RUN pip install --no-cache-dir torch torchvision torchaudio --index-url https://
     pip install --no-cache-dir pytorch-lightning==1.9 --no-deps && \
     pip install --no-cache-dir gdown
 
-RUN conda init bash && \
-    echo "conda activate splat-slam" >> ~/.bashrc
+FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-runtime
+ENV DEBIAN_FRONTEND=noninteractive
 
-COPY download_pretrained_model.sh /usr/local/bin/download_pretrained_model.sh
+RUN apt update && \
+    apt install -y --no-install-recommends tzdata git unzip wget libglib2.0-0 libgl1-mesa-glx && \
+    apt clean && rm -rf /var/lib/apt/lists/*
+    
+COPY --from=builder /workspace/Splat-SLAM /workspace/Splat-SLAM
+COPY --from=builder /opt/conda /opt/conda
+COPY scripts/download_pretrained_model.sh /usr/local/bin/download_pretrained_model.sh
```

Comment:
`COPY download_pretrained_model.sh /usr/local/bin/download_pretrained_model.sh`
or
`git mv download_pretrained_model.sh scripts/download_pretrained_model.sh`

Reply:
Oh great, thanks! Forgot to stage that change. I moved the script to the scripts folder